features/step_definitions/cucumber_rails_steps.rb

```diff
@@ -65,6 +65,8 @@
   }
 
   step 'I append to "features/support/env.rb" with:', selenium_config
+  step 'I append to "Gemfile" with:', "gem 'webrick', group: 'test'\n"
+  run_command_and_stop('bundle install --jobs 4')
```

Comment:
Install the `webrick` gem in this step to support the optional Capybara configuration above.

https://github.com/cucumber/cucumber-rails/blob/3f332f92c030de6c39ba7f2d3e526d16fca45905/features/step_definitions/cucumber_rails_steps.rb#L64